bin/remove_field.py

```diff
@@ -29,15 +29,13 @@
 
 from botocore.client import ClientError
 import click
-from configman import ConfigurationManager
-from configman.environment import environment
 from elasticsearch.exceptions import ConnectionError
 from elasticsearch_dsl import Search
 from more_itertools import chunked
 
-from socorro.external.boto.connection_context import S3Connection
+from socorro import settings
 from socorro.external.boto.crashstorage import build_keys, dict_to_str
-from socorro.external.es.connection_context import ConnectionContext
+from socorro.libclass import build_instance_from_settings
```

Comment:
I thought about putting `libclass.py` into `socorro/lib/`, but then put it one directory up because in bug 1810238, I'm working on reducing redundancy in structure names and paths. In that bug, I'll either move this to `socorro/lib/` or move everything out of `socorro/lib/` up a directory.